uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/LandingScreen.kt

```diff
@@ -28,3 +32,13 @@ fun LandingScreen(modifier: Modifier = Modifier, onTimeout: () -> Unit) {
         )
     }
 }
+
+@Preview(showBackground = true)
+@Composable
+fun PreviewFeedScreen() {
+    ConferenceAppFeederTheme(false) {
+        ProvideFeedViewModel(viewModel = fakeFeedViewModel()) {
```

Comment:
I think `ProvideFeedViewModel` is not needed.
Because `LandingScreen` doesn't require `FeedViewModel` 👍